### PR TITLE
bugfix - not working on dogwood

### PR DESCRIPTION
**Pull request by dadasoz-cuelogic**
1. Bug-fix : not working on dogwood
2. Moved "DEFAULT_SCORM_PATH" outside to edx-platform directory.
- The reason is edx-platform contains the main platform code.
- The best practice for external data is to keep it outside of edx-platform

### CODE REVIEW

File: xb_scorm/xb_scorm.py [hunk 1]
```diff
@@ -65,13 +65,17 @@ def max_score(self):
     tc_statements = Dict(default={}, scope=Scope.user_state)  # statementId -> JSON document
 
     def __init__(self, *args, **kwargs):
-        XBlock.__init__(self, *args, **kwargs)
+        try:
```
Comment (ihowson) [inline xb_scorm/xb_scorm.py]: Is this the only change for Dogwood? Why is it necessary?

Comment (dadasoz-cuelogic) [inline xb_scorm/xb_scorm.py]: Yes, they have upgraded xmodule version, so when working with dogwood getting errors.

`'SCORMXBlockWithMixins' object has no attribute 'xmodule_runtime'`

Comment (dadasoz-cuelogic) [inline xb_scorm/xb_scorm.py]: Removed **init** method,

http://edx-code.narkive.com/6bsrps1W/xblock-overriding-the-init-method

**Comment by dadasoz-cuelogic** [comment]
Issue #2  Fixed

**Comment by dadasoz-cuelogic** [comment]
@ihowson can you please look into this?

**Pull request merged**